tools/wasm_test_runner/src/main.rs

```diff
@@ -0,0 +1,82 @@
+//! A tool to run `WebAssembly` tests
+//!
+//! This copies functionality of `webassembly-test-runner`, but with an ability to indicate failure with an exit code.
+
+use std::process::ExitCode;
+
+use anyhow::{bail, Result};
```

Comment:
Have you considered to use `wasmtime::Result` and not to depend on `anyhow` in this case?

Reply:
`wasmtime::Result` is pricesely the `anyhow::Result`, so I don't think it's beneficial